openmc/tallies.py

```diff
@@ -21,9 +22,58 @@ def reset_auto_tally_id():
 
 
 class Tally(object):
+    """A tally defined by a set of scores that are accumulated for a list of
+    nuclides given a set of filters.
+
+    Parameters
+    ----------
+    tally_id : int
```

Comment:
Well it's not auto-generated but it is not required.
On Jun 24, 2015 4:53 PM, "Paul Romano" <notifications@github.com> wrote:

> In openmc/tallies.py
> <https://github.com/mit-crpg/openmc/pull/400#discussion_r33210957>:
>
> > @@ -21,9 +22,58 @@ def reset_auto_tally_id():
> >
> >
> >  class Tally(object):
> > +    """A tally defined by a set of scores that are accumulated for a list of
> > +    nuclides given a set of filters.
> > +
> > +    Parameters
> > +    ----------
> > +    tally_id : int
>
> I see that the id is autogenerated in the property setter, but I don't see
> where the name is autogenerated. Is it part of the Tally class or elsewhere?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-crpg/openmc/pull/400/files#r33210957>.
>